src/core/include/ngraph/validation_util.hpp

```diff
@@ -187,6 +202,7 @@ namespace opset1 {
 /// \param      pads_begin        The placeholder for paddings at the beginning of axis.
 /// \param      pads_end          The placeholder for paddings at the end of axis.
 ///
+OPENVINO_DEPRECATED("This function is deprecated and will be moved to dev api in 2024.0 release.")
```

Comment:
what are the replacements for all these functions?

Reply:
There are no replacement.
We just plan to move them from public api to dev api in 2024.0

Reply:
In general you had to deprecate only API from openvino folder... This API anyway will be removed in 2024.0... And we will deprecate all legacy API in the next release. :)